scripts/simtest/cargo-simtest

```diff
@@ -87,35 +85,22 @@ if ! cargo nextest --help > /dev/null 2>&1; then
   exit 1
 fi
 
-if [ -n "$1" ]; then
-  echo "Running test $1"
-  CARGO_COMMAND=(nextest run "$1" --cargo-profile simulator --run-ignored all)
-else
-  echo "Running all simtest"
-  # Adding `simtest` after `run` since all walrus simtest is prefixed with `simtest`.
-  CARGO_COMMAND=(nextest run simtest --cargo-profile simulator --run-ignored all)
-fi
-
-if [ "$1" = "build" ]; then
+cargo_command=( nextest run --cargo-profile simulator )
+if [[ "$1" = "build" ]]; then
   shift
-  CARGO_COMMAND=(build --profile simulator)
+  cargo_command=( build --profile simulator )
 fi
 
 # Must supply a new temp dir - the test is deterministic and can't choose one randomly itself.
-export TMPDIR=$(mktemp -d)
+TMPDIR="$(mktemp -d)"
+export TMPDIR
 
 # Set the example move package for the simtest static initializer
 # https://github.com/MystenLabs/sui/blob/7bc276d534c6c758ac2cfefe96431c2b1318ca01/crates/sui-proc-macros/src/lib.rs#L52
-root_dir="$(git rev-parse --show-toplevel)"
-export SIMTEST_STATIC_INIT_MOVE="$root_dir/contracts/walrus"
+root_dir=$(git rev-parse --show-toplevel)
+export SIMTEST_STATIC_INIT_MOVE=$root_dir"/examples/move/basics"
```

Comment:
Important: This has to be reverted.

Reply:
Gotcha. I'll take a look. I was wondering why this was failing. 
Good catch!